### PR TITLE
Show stringified extras in dataset modal

### PR DESCRIPTION
**Pull request by abulte**
Assuming `JSON.stringify` is safe enough since this can be a user provided value 🧐. 

## Before

Capture d’écran 2020-08-25 à 23 25 47

## After

Capture d’écran 2020-08-25 à 23 28 48

### CODE REVIEW

**Comment by abulte** [comment]
@AntoineAugusti (can't assign you a review)

File: js/front/dataset/details-modal.vue [hunk 1]
```diff
@@ -57,6 +57,14 @@ export default {
     props: {
         dataset: Object
     },
-    components: {Modal}
+    components: {Modal},
+    methods: {
+        stringify(value) {
+            if (typeof value === 'string' || value instanceof String) {
+                return value;
+            }
```
Comment (AntoineAugusti) [inline js/front/dataset/details-modal.vue:65]: We can have floats and integers if I remember. Did you try testing if it's an object?

It seems it's not pretty to write the check https://stackoverflow.com/a/16608045

File: js/front/dataset/details-modal.vue [hunk 2]
```diff
@@ -57,6 +57,14 @@ export default {
...
+            return JSON.stringify(value);
```
Comment (AntoineAugusti) [inline js/front/dataset/details-modal.vue:66]: From my understanding this + `{{ }}` is XSS safe 👮‍♀️

**Comment by ThibaudDauce** [comment]
I think it's done :-)

**Pull request closed without merging**